app/src/main/java/com/github/libretube/ui/fragments/PlayerFragment.kt

```diff
@@ -463,10 +463,20 @@ class PlayerFragment : BaseFragment(), OnlinePlayerOptions {
             // hide the description and chapters
             binding.playerDescriptionArrow.animate().rotation(0F).setDuration(250).start()
             binding.descLinLayout.visibility = View.GONE
+
+            // show formated short view count
+            val viewInfo = context?.getString(R.string.views, streams.views.formatShort()) +
+                if (!isLive) TextUtils.SEPARATOR + streams.uploadDate else ""
```

Comment:
Instead of using that line twice we might just want to save that text to a variable before the if/else condition and reuse it in the if and the else clause. Apart from that `getString()` should be sufficient, using `context?#getString()!!` shouldn't be needed as fragments implement their own method for it.

Reply:
not sure then how to use `streams.views.formatShort()` and `String.format("%,d", streams.views)` in if else than.

> Apart from that getString() should be sufficient, using context?#getString()!! shouldn't be needed as fragments implement their own method for it.

changed that 👍🏼

Reply:
> not sure then how to use `streams.views.formatShort()` and `String.format("%,d", streams.views)` in if else than.
>
I only meant saving `if (!isLive) TextUtils.SEPARATOR + streams.uploadDate else ""` to a variable before (just to avoid duplicating it) and later adding it to the end of the formatted views string.
The string formatting has to be done each time individually as it's already done in your PR.

Reply:
ohk, got it.